sample_data.csv

```diff
@@ -4998,4 +4998,4 @@ id,created_on,updated_on,identifier,provider,source,foreign_identifier,foreign_l
+10494466,2018-09-11 13:06:48.062657+00,2018-10-19 14:20:07.92795+00,97f9ee0a-4f08-4027-85ca-a60e2f395705,behance,commoncrawl,https://mir-s3-cdn-cf.behance.net/project_modules/max_1200/613aa927443639.56050a60576fb.jpg,https://www.behance.net/gallery/3548599/KBF,https://mir-s3-cdn-cf.behance.net/project_modules/max_1200/613aa927443639.56050a60576fb.jpg,,,,,by-nc-nd,4.0,"Teren Prywatny, Anna Maria Bojarowicz",,KBF,,2018-09-11 13:06:48.062657+00,True,"{""set"": ""https://www.behance.net/gallery/3548599/KBF"", ""description"": ""Projekt publikacji powsta\\u0142 w ramach obchod\\xf3w 70. Rocznicy wybuchu II Wojny \\u015awiatowej i wr\\u0119czenia medali \""Bohaterom Wrze\\u015bnia 1939\"".|| The project was created for the 70th anniversary of the start of the Second World War and the ceremony of the honoring r&#8230;""}","[{""name"": ""70.rocznica drugiej wojny"", ""provider"": ""behance""}, {""name"": ""suchy tłok"", ""provider"": ""behance""}, {""name"": ""okładka"", ""provider"": ""behance""}, {""name"": ""cover"", ""provider"": ""behance""}, {""name"": ""Adobe Portfolio"", ""provider"": ""behance""}, {""name"": ""KBF"", ""provider"": ""behance""}, {""name"": ""Anna Bojarowicz"", ""provider"": ""behance""}, {""name"": ""marcin przybyłko"", ""provider"": ""behance""}, {""name"": ""book design"", ""provider"": ""behance""}, {""name"": ""2nd world war"", ""provider"": ""behance""}, {""name"": ""Druga wojna światowa"", ""provider"": ""behance""}, {""name"": ""paper"", ""accuracy"": 0.97929, ""provider"": ""clarifai""}, {""name"": ""education"", ""accuracy"": 0.96782, ""provider"": ""clarifai""}, {""name"": ""literature"", ""accuracy"": 0.95893, ""provider"": ""clarifai""}, {""name"": ""no person"", ""accuracy"": 0.95874, ""provider"": ""clarifai""}, {""name"": ""book bindings"", ""accuracy"": 0.9537, ""provider"": ""clarifai""}, {""name"": ""page"", ""accuracy"": 0.95302, ""provider"": ""clarifai""}, {""name"": ""knowledge"", ""accuracy"": 0.92456, ""provider"": ""clarifai""}, {""name"": ""empty"", ""accuracy"": 0.91178, ""provider"": ""clarifai""}, {""name"": ""college"", ""accuracy"": 0.90942, ""provider"": ""clarifai""}, {""name"": ""facts"", ""accuracy"": 0.9042, ""provider"": ""clarifai""}, {""name"": ""document"", ""accuracy"": 0.90386, ""provider"": ""clarifai""}, {""name"": ""devoid"", ""accuracy"": 0.89191, ""provider"": ""clarifai""}, {""name"": ""writing"", ""accuracy"": 0.88764, ""provider"": ""clarifai""}, {""name"": ""textbook"", ""accuracy"": 0.8857, ""provider"": ""clarifai""}, {""name"": ""cover"", ""accuracy"": 0.88136, ""provider"": ""clarifai""}, {""name"": ""school"", ""accuracy"": 0.87721, ""provider"": ""clarifai""}, {""name"": ""blank"", ""accuracy"": 0.87476, ""provider"": ""clarifai""}, {""name"": ""library"", ""accuracy"": 0.87286, ""provider"": ""clarifai""}, {""name"": ""write"", ""accuracy"": 0.87247, ""provider"": ""clarifai""}, {""name"": ""business"", ""accuracy"": 0.84831, ""provider"": ""clarifai""}]",f,0
```

Comment:
Going forward it's better to insert new data instead of editing sample data (as other tests may depend on those particular documents). That doesn't apply to this particular case, but it's a good thing to keep in mind (no action required, just hit the "resolve" button when you see this comment)

Reply:
Yes, I agree. I tried it first to see if any test failed because of this change, and when it didn't decided to keep it this way. The decision was also partly motivated by not being sure of how to add unique values for some of the fields correctly. But I'll keep this in mind.